### DISCUSSION TITLE
请问如何在 fcitx-table-extra 的 98 五笔中关闭调频功能？

### DISCUSSION THREAD

**Discussion by YisuiDenghua** [fcitx/fcitx5#857] [Q&A] [answered]
请问如何在 fcitx-tables-extra 的 98 五笔中关闭调频功能？我阅读了 fcitx-tables-extra 的 readme，issue 与 Fcitx5 Wiki，但没有找到该问题的解决方法。

**Comment by wengxt** [comment, marked as answer]
设置 -> 排序规则 -> 否
